cli/args/mod.rs

```diff
@@ -1129,18 +1129,7 @@ impl CliOptions {
       let all_valid_unstable_flags: Vec<&str> = crate::UNSTABLE_GRANULAR_FLAGS
         .iter()
         .map(|granular_flag| granular_flag.name)
-        .chain([
-          "byonm",
-          "bare-node-builtins",
-          "detect-cjs",
-          "fmt-component",
-          "fmt-sql",
-          "lazy-dynamic-imports",
-          "npm-lazy-caching",
-          "npm-patch",
-          "sloppy-imports",
-          "lockfile-v5",
-        ])
+        .chain(["fmt-component", "fmt-sql", "npm-lazy-caching", "npm-patch"])
```

Comment:
These few were never defined as unstable flags (ie. only available as options to `unstable` key in `deno.jsonc` file). We should decide if we want to make them flags.

Reply:
I will address it in a follow up PR